src/app/admin-view/admin-forms/services/create-common-project.service.ts

```diff
@@ -0,0 +1,55 @@
+import {FormGroup} from '@angular/forms';
+import {HttpClient} from '@angular/common/http';
+import {Injectable} from '@angular/core';
+import {ProjectData} from '../../../models/project-data';
+import {FileService} from '../../services/file-service';
+import {forkJoin, Observable} from 'rxjs';
+import {exhaustMap} from 'rxjs/operators';
+import {ProjectsService} from '../../../services/projects-service';
+
+@Injectable()
+export class CreateCommonProjectService {
+
+  private requestEntity = {} as ProjectData;
+
+  constructor(private httpClient: HttpClient, private fileService: FileService, private projectService: ProjectsService) {
+  }
+
+  public createProject(form: FormGroup, category: string): Observable<any> {
```

Comment:
any Im not very happy with that dal by som tam nejaky interface alebo primitivny datovy typ podla toho co vracia back